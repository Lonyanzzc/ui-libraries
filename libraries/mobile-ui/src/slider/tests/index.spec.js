import {
  describe,
  it,
  test,
  vi as jest,
  expect,
} from 'vitest';
  import Slider from '..';
import {
  mount,
} from '../../../test';


test('slider-width style', () => {
  const wrapper = mount(Slider, {
    propsData: {
      style: "width: 100px; height: 50px; padding: 10px;",
    },
  });

  expect(wrapper.html()).toMatchSnapshot();
});
