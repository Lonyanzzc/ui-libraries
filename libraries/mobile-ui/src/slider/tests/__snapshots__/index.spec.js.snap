// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`slider-width style 1`] = `
<div class="van-slider-room " style="width: 100px; height: 50px; padding: 10px;">
  <div class="van-slider">
    <div class="van-slider__bar" style="width: 0%;">
      <div role="slider" tabindex="0" aria-valuemin="0" aria-valuenow="0" aria-valuemax="100" aria-orientation="horizontal" class="van-slider__button-wrapper">
        <div class="van-slider__button"></div>
      </div>
    </div>
  </div>
</div>
`;
